src/SprykerEco/Zed/Computop/Communication/Plugin/Checkout/ComputopPostCheckPlugin.php

```diff
@@ -9,17 +9,20 @@
 
 use Generated\Shared\Transfer\CheckoutResponseTransfer;
 use Generated\Shared\Transfer\QuoteTransfer;
+use Spryker\Zed\Checkout\Dependency\Plugin\CheckoutPostSaveHookInterface;
 use Spryker\Zed\Kernel\Communication\AbstractPlugin;
-use Spryker\Zed\Payment\Dependency\Plugin\Checkout\CheckoutPluginInterface;
 
 /**
  * @method \SprykerEco\Zed\Computop\Business\ComputopFacadeInterface getFacade()
  * @method \SprykerEco\Zed\Computop\Communication\ComputopCommunicationFactory getFactory()
+ * @method \SprykerEco\Zed\Computop\ComputopConfig getConfig()
+ * @method \SprykerEco\Zed\Computop\Persistence\ComputopQueryContainerInterface getQueryContainer()
  */
-class ComputopPostCheckPlugin extends AbstractPlugin implements CheckoutPluginInterface
+class ComputopPostCheckPlugin extends AbstractPlugin implements CheckoutPostSaveHookInterface
 {
     /**
-     * This plugin is called after the order is placed.
+     * Specification:
```

Comment:
Usually we should have it like {@inheritdoc} and plugin specific info on the next row